lib/src/model/package_graph.dart

```diff
@@ -554,7 +554,12 @@ class PackageGraph with CommentReferable, Nameable {
     alreadyTagged.add(key);
     // Mark that `publicLibrary` exports `libraryElement`.
     _libraryExports.putIfAbsent(libraryElement, () => {}).add(publicLibrary);
-    for (var exportedElement in libraryElement.libraryExports) {
+
+    var exported = [
+      ...libraryElement.libraryExports,
```

Comment:
Theoretically units provide all the exports and imports that can be in the library.

Reply:
Ah it looks like you're right; I didn't test this with new unit tests, but they look good. Thanks